tests/ballerina-unit-test/src/test/resources/test-src/expressions/stamp/anydata-stamp-expr-test.bal

```diff
@@ -0,0 +1,93 @@
+type Employee record {
+    string name;
+    string status;
+    string batch;
+};
+
+type Teacher record {
+    string name;
+    int age;
+    string status;
+    string batch;
+    string school;
+};
+
+function stampAnydataToJSON() returns json|error  {
+
+    anydata anydataValue = 3;
+    json|error  jsonValue = json.stamp(anydataValue);
+
+    return jsonValue;
+}
+
+function stampAnydataToRecord() returns Employee|error  {
+    Teacher t1 = { name: "Raja", age: 25, status: "single", batch: "LK2014", school: "Hindu College" };
+    anydata anydataValue = t1;
+    Employee|error  employee = Employee.stamp(anydataValue);
+    return employee;
+}
+
+function stampAnydataToJSONV2() returns json|error  {
+    json t1 = { name: "Raja", age: 25, status: "single", batch: "LK2014", school: "Hindu College" };
+    anydata anydataValue = t1;
+
+    json|error  jsonValue = json.stamp(anydataValue);
+    return jsonValue;
+}
+
+function stampAnydataToXML() returns xml|error  {
+
+    anydata anydataValue = xml `<book>The Lost World</book>`;
+
+    xml|error  xmlValue = xml.stamp(anydataValue);
+    return xmlValue;
+}
+
+function stampAnydataToMap() returns map<Employee>|error  {
+    Teacher p1 = { name: "Raja", age: 25, status: "single", batch: "LK2014", school: "Hindu College" };
+    Teacher p2 = { name: "Mohan", age: 30, status: "single", batch: "LK2014", school: "Hindu College" };
+
+    map<Teacher> teacherMap = { "a": p1, "b": p2 };
+    anydata anydataValue = teacherMap;
+    map<Employee>|error  mapValue = map<Employee>.stamp(anydataValue);
+
+    return mapValue;
+}
+
+function stampAnydataToRecordArray() returns Teacher[]|error  {
+
+    Teacher p1 = { name: "Raja", age: 25, status: "single", batch: "LK2014", school: "Hindu College" };
+    Teacher p2 = { name: "Mohan", age: 30, status: "single", batch: "LK2014", school: "Hindu College" };
+
+    Teacher[] teacherArray = [p1, p2];
+    anydata anydataValue = teacherArray;
+    Teacher[]|error  returnValue = Teacher[].stamp(anydataValue);
+
+    return returnValue;
+}
+
+function stampAnydataToTuple() returns (string,Teacher)|error  {
+
+    (string, Teacher)  tupleValue = ("Mohan", { name: "Raja", age: 25, status: "single", batch: "LK2014", school:
+    "Hindu College" });
+
+    anydata anydataValue = tupleValue;
+    (string,Teacher)|error  returnValue = (string, Teacher).stamp(anydataValue);
+
+    return returnValue;
+}
+
+function stampAnydataMapToAnyMap() returns map<anydata> {
+    map<anydata> anydataMap = { name: "Raja", age: 25, status: "single", batch: "LK2014", school: "Hindu College" };
+
+    map<anydata> mapValue = map<anydata>.stamp(anydataMap);
+    return mapValue;
+}
+
+function stampAnydataToAnydata() returns anydata {
+    json jsonValue = { name: "Raja", age: 25, status: "single", batch: "LK2014", school: "Hindu College" };
+    anydata anydataValue = jsonValue;
+    anydata returnValue = anydata.stamp(anydataValue);
+
+    return returnValue;
+}
```

Comment:
Shall we add a new line to the end of the file?

Reply:
Fixed.